source/PlayerInfo.h

```diff
@@ -60,6 +60,13 @@ class PlayerInfo {
 		int64_t maintenanceCosts = 0;
 		int64_t assetsReturns = 0;
 	};
+
+	enum class TeleportStatus : int {
+		NONE,
```

Comment:
If you're making these correspond to bool NONe should prob explicitely be 0?
(By doing this `NONE: 0`)

Reply:
done